toolchain/check/impl.cpp

```diff
@@ -219,7 +225,7 @@ auto FinishImplWitness(Context& context, SemIR::ImplId impl_id) -> void {
               SemIR::NameId, SemIR::NameId);
           auto builder =
               context.emitter().Build(impl.definition_id, ImplMissingFunction,
-                                      fn.name_id, interface.name_id);
+                                      fn.name_id, interface_name_id);
```

Comment:
Regarding `fn.name_id` here, could that be invalidated by `LookupNameInExactScope`, or is it stable in context? (was looking again due to granularity questions)